### PR TITLE
Mobile menu using javaScript

### PR DESCRIPTION
**Pull request by oovillagran**
For this milestone of my portfolio website, I implemented the mobile menu using JavaScript tools, I used the Figma Template number 1 for this project.

The general requirements of this milestone are:
- [ ] No linter errors.
- [ ] Correct use of GitHub flow.
- [ ] Work documented in a professional way.

**Project Requirements**
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

I ask for your kind review in order to continue with the develop of my project.

Thank you for your support.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  });
+
+  popDiv.appendChild(barDiv);
+
+  barDiv.style.width = '30%';
+  barDiv.style.height = '0.5rem';
+  barDiv.style.background = '#000';
+  barDiv.style.marginLeft = '35%';
+  barDiv.style.marginTop = '46rem'
+  // barDiv.style.padding = '50%';
```
Comment (divinecharlotte) [inline script.js]: - [x] kindly consider removing  commented codes it makes your code look messy

File: script.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  popDiv.style.minHeight = '100vh';
+  popDiv.style.opacity = '0.8';
+  popDiv.style.backgroundColor = '#6070FF';
+  uList.style.listStyleType = 'none';
+  headerMenu.style.filter = 'blur(6px)';
+  mSection.style.filter = 'blur(6px)';
+  body.style.overflow = 'hidden';
+    
+  body.insertBefore(popDiv, backing);
+  popDiv.appendChild(uList);
```
Comment (divinecharlotte) [inline script.js]: - [ ] kindly consider removing the the margin left and top to this part check the below image for more clarrification

- your design 👇 
- Screen Shot 2023-03-16 at 18 23 57
- Figma design 👇 
Screen Shot 2023-03-16 at 18 28 44
 this make your design a perfect duplicate of the provided design

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @oovillagran   ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100:
- No linters errors :heavy_check_mark:
- Descriptive commit messages :heavy_check_mark:
- clean code :heavy_check_mark:
- Descriptive PR :heavy_check_mark:



## Required Changes :recycle:
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (oovillagran) [reply script.js]: Thank you @divinecharlotte, I appreciate your comments!! 👍

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯


## Highlights 🔥🔥🔥🔥
- Followed GitHub flow ✅
- No Linters error 🤩🔥💯
- Great working on the functionality and design 🔥🔥
- Professional README 🔥✨💯💯
- Descriptive PR 🤩💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thank you so much @Reem-lab!!!

**Pull request merged**